changelog/1011.feature.rst

```diff
@@ -0,0 +1,4 @@
+Allow editing Onboarding.
+    - :class:`OnboardingPrompt` and :class:`OnboardingPromptOption` are now user-constructable.
+    - Add :meth:`Guild.edit_onboarding`.
+    - Add :class:`OnboardingMode` enum.
```

Comment:
```suggestion
Allow editing Onboarding configurations.
    - :class:`OnboardingPrompt` and :class:`OnboardingPromptOption` are now user-constructible.
    - Add :meth:`Guild.edit_onboarding`.
    - Add :attr:`Onboarding.mode`.
```
(constructable vs constructible is a minor nit, I believe both are technically correct but the latter is more common)